package.json

```diff
@@ -100,22 +100,23 @@
     "compression": "^1.7.0",
     "connect-history-api-fallback": "^1.3.0",
     "express": "^4.15.3",
-    "long": "^3.2.0",
+    "long": "^4.0.0",
     "minimist": "^1.2.0",
-    "mobx": "^3.3.1",
-    "mobx-react": "^4.3.3",
+    "mobx": "^4.1.0",
+    "mobx-react": "^5.0.0",
+    "mobx-utils": "^4.0.0",
     "nuclearnet.js": "^1.5.2",
     "protobufjs": "6.7.3",
     "react": "^16.2.0",
     "react-color": "^2.13.8",
     "react-dom": "^16.0.0",
-    "react-resize-detector": "^1.1.0",
+    "react-resize-detector": "^2.2.0",
     "react-router": "^4.2.0",
     "react-router-dom": "^4.2.2",
     "serve-favicon": "^2.4.3",
-    "socket.io": "^1.7.3",
-    "socket.io-client": "^1.7.3",
-    "three": "^0.84.0",
+    "socket.io": "^2.0.4",
+    "socket.io-client": "^2.0.4",
+    "three": "^0.91.0",
```

Comment:
The types are only for `0.89.12`.

Reply:
That's as high as they go :(
But I don't think we would use any of the bleeding edge features of ThreeJS. (not that we could with outdated types)

Reply:
Seems this one might have been a problem though :P

Reply:
Nevermind, it's fine we just accidentally deleted some code we shouldn't

Reply:
> But I don't think we would use any of the bleeding edge features of ThreeJS.

That would indicate we should downgrade three.js to match the types.

Reply:
That is a valid point. We may have been overzealous with this PR :P